### PR TITLE
Key Features page update (PDFNET-57965)

### PR DESCRIPTION
**Pull request by AlexBiriukov**
Key Features page update

### CODE REVIEW

**Review by andruhovski — APPROVED** [review]
No objections

File: net/get-started/key-features/_index.md [hunk 1]
```diff
@@ -13,20 +13,49 @@ sitemap:
...
+Aspose.PDF provides powerful Document Object Model (DOM) to explore and modify all types of PDF content. Supports loading from and saving to a wide list of [file formats](/net/supported-file-formats/). 
 
 ## Supported PDF versions
 
-Aspose.PDF for .NET supports PDF versions 1.2, 1.3, 1.4, 1.5, 1.6 and 1.7.
+Aspose.PDF for .NET supports PDF versions 1.2, 1.3, 1.4, 1.5, 1.6, 1.7 and 2.0.
+
+## Document
+
+- Get and Set document viewer properties.
```
[the remark this answers is not in the document]
Comment (AlexBiriukov) [reply net/get-started/key-features/_index.md]: Accepted

File: net/get-started/key-features/_index.md [hunk 2]
```diff
@@ -104,50 +99,37 @@ Aspose.PDF for .NET supports PDF versions 1.2, 1.3, 1.4, 1.5, 1.6 and 1.7.
...
 
-- Create application link.
-- Create JavaScript link.
-- Remove document open action.
+- Create application, Web, and JavaScript links.
 - Extract links.
+- Remove document open action.
+- Sanitize a PDF document from potentially harmful links and actions.
 
 ## Security and Signatures
```
[the remark this answers is not in the document]
Comment (AlexBiriukov) [reply net/get-started/key-features/_index.md:206]: Accepted

File: net/get-started/key-features/_index.md [hunk 3]
```diff
@@ -104,50 +99,37 @@ Aspose.PDF for .NET supports PDF versions 1.2, 1.3, 1.4, 1.5, 1.6 and 1.7.
...
 
 - Encrypt PDF.
 - Decrypt PDF.
 - Modify password.
 
-## Miscellaneous
+## AI capabilities
 
-Get build version information.
+- Perform popular or personalized queries to a PDF document with your chatGPT or Llama account.
```
[the remark this answers is not in the document]
Comment (AlexBiriukov) [reply net/get-started/key-features/_index.md]: Accepted

File: net/get-started/key-features/_index.md [hunk 4]
```diff
@@ -104,50 +99,37 @@ Aspose.PDF for .NET supports PDF versions 1.2, 1.3, 1.4, 1.5, 1.6 and 1.7.
...
+- Sanitize a PDF document from potentially harmful links and actions.
 
 ## Security and Signatures
 
 - Encrypt PDF.
 - Decrypt PDF.
 - Modify password.
 
-## Miscellaneous
+## AI capabilities
```
[the remark this answers is not in the document]
Comment (AlexBiriukov) [reply net/get-started/key-features/_index.md]: Accepted

**Review by andruhovski — APPROVED** [review]

**Pull request merged**